docs/proposals/docker/docker_network_module.md

```diff
@@ -0,0 +1,130 @@
+# Docker_Network Module Proposal
+
+## Purpose and Scope:
+
+The purpose of Docker_network is to create networks, connect containers to networks, disconnect containers from 
+networks, and delete networks.
+
+Docker network will manage networks using docker-py to communicate with either a local or remote API. It will
+support API versions >= 1.14. API connection details will be handled externally in a shared utility module similar to
+how other cloud modules operate.
+
+## Parameters:
+
+Docker_network will accept the parameters listed below. Parameters related to connecting to the API will be handled in 
+a shared utility module, as mentioned above.
+
+```
+containers:
+
+network_name:
+  description:
+    - Name of the network to operate on.
+  default: null
+  required: true
+
+driver:
+  description:
+    - Specify the type of network. Docker provides bridge and overlay drivers, but 3rd party drivers can also be used.
+  default: bridge
+
+options:
+  description:
+    - Dictionary of network settings. Consult docker docs for valid options and values.
+  default: null
+
+connected:
+  description:
+    - List of container names or container IDs to connect to a network.
+  default: null
+  
+disconnected:
+  description:
+    - List of container names or container IDs to disconnect from a network.
+  default: null
```

Comment:
i would argue that connected should be enough, any ids not in that list should be automatically disconnected

possibly with an `incremental=no|yes` option to make it additive vs cannonical